src/pages/City/AddAdministratorModal/AddCitiesSecretaryForm.less

```diff
@@ -0,0 +1,11 @@
+.ant-select {
+  max-width: 310px;
+}
+.ant-picker {
+  width: 310px;
+}
+
+.formItemExplain {
+  width: 310px;
+  float: right;
+}
```

Comment:
Add an empty row at the end of the file please)

Reply:
Done